### DISCUSSION TITLE
Admin Account Merged with Client and Admin no longer available

### DISCUSSION THREAD

**Discussion by RaulBustamante** [ltsp/ltsp#602] [Q&A] [answered]
I modified the name for my login user (Team1) to my name for testing purposes to "myname" I was logged in the server pc with my new user name "myname" and I also logged in a different pc as a client with the same user name. (which has admin rights)

I changed the background in the client pc, logged out, then logged out of the admin pc and when I logged back in to the admin pc and see if the change was reflected.

Even though I logged in as the admin, LTSP is no longer recognized by client pc's and It seemed to merge the admin as a regular group member.

Still not exactly sure what happened.

Thanks.

**Comment by alkisg** [comment]
Hi @RaulBustamante,

I'm not sure what you're describing, but logging in to multiple places with the same account is not supported by GNOME, KDE, MATE, XFCE all any other desktop environment that I know of.

For example, I've seen the dconf settings database get corrupted because it's not designed to be used simultaneously from different sessions. Then that user couldn't even change the background or any other settings at all.

Another thing to notice is that when you're modifying user accounts on the server, you need to run `sudo ltsp initrd` and reboot the clients for them to take effect.

Feel free to explain things in more detail if these didn't answer your questions, as I'm not sure I understood correctly.

**Comment by RaulBustamante** [comment]
![Error](https://user-images.githubusercontent.com/93734208/140530237-8648d1b3-7b7f-4441-89ac-fa78c200162e.p

Sorry If i am having trouble explaining what the issue is, I recently installed LTSP, 3 different images were created and it seemed to be working correctly. I made some user changes, actually ran 
sudo ltsp initrd, then it started these issues. Now when booting any Client PC it is providing the following error: 

<pre>root@tcartserver:/srv/tftp/ltsp# systemctl status dnsmasq.service 
<font color="#CC0000"><b>●</b></font> dnsmasq.service - dnsmasq - A lightweight DHCP and caching DNS server
     Loaded: loaded (/lib/systemd/system/dnsmasq.service; enabled; vendor prese<span style="background-color:#FFFFFF"><font color="#300A24">&gt;</font></span>
     Active: <font color="#CC0000"><b>failed</b></font> (Result: exit-code) since Thu 2021-11-04 19:32:49 EDT; 29mi<span style="background-color:#FFFFFF"><font color="#300A24">&gt;</font></span>

Nov 04 19:32:49 tcartserver systemd[1]: Starting dnsmasq - A lightweight DHCP a<span style="background-color:#FFFFFF"><font color="#300A24">&gt;</font></span>
Nov 04 19:32:49 tcartserver dnsmasq[632]: dnsmasq: syntax check OK.
Nov 04 19:32:49 tcartserver dnsmasq[655]: dnsmasq: TFTP directory /srv/tftp ina<span style="background-color:#FFFFFF"><font color="#300A24">&gt;</font></span>
Nov 04 19:32:49 tcartserver systemd[1]: <b>dnsmasq.service: Control process exited</b><span style="background-color:#FFFFFF"><font color="#300A24">&gt;</font></span>
Nov 04 19:32:49 tcartserver dnsmasq[655]: <font color="#CC0000"><b>TFTP directory /srv/tftp inaccessible</b></font><span style="background-color:#FFFFFF"><font color="#300A24">&gt;</font></span>
Nov 04 19:32:49 tcartserver systemd[1]: <font color="#D7D75F"><b>dnsmasq.service: Failed with result &apos;ex</b></font><span style="background-color:#FFFFFF"><font color="#300A24">&gt;</font></span>
Nov 04 19:32:49 tcartserver dnsmasq[655]: <font color="#CC0000"><b>FAILED to start up</b></font>
Nov 04 19:32:49 tcartserver systemd[1]: <font color="#CC0000"><b>Failed to start dnsmasq - A lightweight</b></font><span style="background-color:#FFFFFF"><font color="#300A24">&gt;</font></span>
root@tcartserver:/srv/tftp/ltsp# enable-tftp
</pre>

**Comment by alkisg** [comment, marked as answer]
As the message says, your /srv/tftp directory has access permissions. Fix them so that "ls /srv/tftp" works from a non-root account.

**Reply by RaulBustamante** [reply]
Thank you! I had to run the following commands again and it worked: 
ltsp dnsmasq 
ltsp nfs 
ltsp ipxe
